docs/advanced/index.rst

```diff
@@ -20,3 +20,4 @@ Various advanced topics regarding cookiecutter usage.
    choice_variables
    dict_variables
    template_extensions
+   subdirectories
```

Comment:
```suggestion
   repo_subdirectories
```
Do not bring new errors to docs :)

P.S. I know that docs has some not-rendered endpoints right now. Such errors already fixed at #1216